results-2021.md

```diff
@@ -0,0 +1,461 @@
+# Government frontend survey results 2021
+
+The 2021 Frontend survey asks additional questions in a different order from the previous surveys, [2016](./results-2016.md) and [2018](./results-2018.md). For this survey, we are not interested in prototyping projects and those responses are not included in the results.
+
+Respondents can skip any question and where users do skip, it's recorded as 'No answer'.
+
+## Q1. Thinking about your current project, how would you describe it? If you work on more than one project, pick the one you spend the most time on. Keep this project in mind for the rest of the survey.
```

Comment:
This heading has a different structure to the others, it starts with 'Q1' instead of 'Question 1', is this intentional?

Reply:
I missed this one. Have changed it now.